### PR TITLE
PS-10209: Configurable bundle RFQ and "Quantity in the Cart" (phase 1).

### PR DESCRIPTION
**Pull request by dmiseev**
Branch: backport/ps-10209/cart-5.11.0
Ticket: https://spryker.atlassian.net/browse/PS-10209
Target Version: 5.11.0

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Cart               | minor                 |                       |

#### Release Notes

{skip}

-----------------------------------------

##### Change log

Improvements
- Introduced `CartClient::addToCart()`.
- Introduced `CartClient::removeFromCart()`.
- Introduced `CartClient::updateQuantity()`.
- Introduced `CartFacade::removeFromCart()`.
- `SessionQuoteStorageStrategyPlugin` plugin implements `CartOperationQuoteStorageStrategyPluginInterface`.

Deprecations
- Deprecated `PostSavePluginInterface`.

### CODE REVIEW

File: src/Spryker/Client/Cart/Plugin/SessionQuoteStorageStrategyPlugin.php [hunk 1]
```diff
@@ -206,6 +207,60 @@ public function changeItemQuantity($sku, $groupKey = null, $quantity = 1)
         return $this->increaseItemQuantity($sku, $groupKey, $delta);
     }
 
+    /**
+     * Specification:
+     * - Makes zed request.
```
Comment (dereuromark) [inline src/Spryker/Client/Cart/Plugin/SessionQuoteStorageStrategyPlugin.php:212]: Zed
please also check other occurrences.

Comment (dereuromark) [reply src/Spryker/Client/Cart/Plugin/SessionQuoteStorageStrategyPlugin.php:212]: Backport is not that important, please check this on the main master releases.

Comment (dmiseev) [reply src/Spryker/Client/Cart/Plugin/SessionQuoteStorageStrategyPlugin.php:212]: ok, thx

**Comment by gerner-spryker** [comment]
Info: base branch verified

**Review by gerner-spryker — APPROVED** [review]

**Comment by dereuromark** [comment]
Release: https://release.spryker.com/release/hotfix?pr=https%3A%2F%2Fgithub.com%2Fspryker%2Fcart%2Fpull%2F4

**Pull request closed without merging**